.github/workflows/main.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build-nix:
+    name: Ubuntu xenial (x86_64) debug
+    runs-on: ubuntu-16.04
+    env:
+      BUILD_TYPE: Debug
```

Comment:
Consider using `BUILD_TYPE: Release` as a global env and skip the Debug here, it's pretty much useless

Reply:
I'm not an expert with regards to GitHub actions. Mind to push the necessary changes?

Reply:
Well, actually, it makes sense to build things not just with the release config on the CI, but also with `Debug` as a target. Most of us use the latter locally for development, and the former is used in our release builds.